easybuild/easyblocks/j/jaxlib.py

```diff
@@ -69,13 +80,18 @@ def configure_step(self):
 
         # Collect options for the build script
         # Used only by the build script
-        options = [
-            '--target_cpu_features=default',  # Using copt for optimizations
-        ]
+
+        # C++ flags are set through copt below
+        if build_option('optarch') == OPTARCH_GENERIC:
+            options = ['--target_cpu_features=default']
+        else:
+            options = ['--target_cpu_features=native']
```

Comment:
@lexming Doesn't this mean that `jaxlib` will no longer be built with the compiler options we specify in `$CXXFLAGS`?

Reply:
The rest of `CXXFLAGS` are respected, `native` only adds `--copt=-march=native` in Bazel. But now I just realised that `-march=native` will be already caught into the `copt` parameters set below, so this setting is redundant. I'll remove it.

Reply:
done in https://github.com/easybuilders/easybuild-easyblocks/pull/2619/commits/e11b2df7ee533b49ac6ed28c5e64d62d3fb53131